experiments/experiments.go

```diff
@@ -57,6 +57,8 @@ type Experiments struct {
 //
 // Context should come with a timeout otherwise this might block forever, i.e.
 // if the path never becomes available.
+//
+// Deprecated: baseplate.go/experiments is deprecated. Instead, use Decider API (github.snooguts.net/reddit-go/decider).
```

Comment:
it's easier to just mark the whole package as deprecated, see https://github.com/reddit/baseplate.go/blob/b21d4c278843ac83543c84248aaced67aec7a06d/internalv2compat/compat.go#L6 for an example.

also I don't think we want to reference internal ghe from the open source code.

Reply:
ooh great catch, thanks!
And you're right, let me remove the reference.

Reply:
@fishy could you advisee on a separate question:
@kylelemons gave a great suggestion of putting in a prometheus metric into these deprecated `Variant()`/`Expose()` functions so we can see any stragglers still using the old code.

Could you point to an example of how I might be able to add those metrics? I'm not too familiar with the BP.go structure much : (

Reply:
@mrlevitas you can check out `prometheus.go` in the `thriftbp` package for some examples: https://github.com/reddit/baseplate.go/blob/b21d4c278843ac83543c84248aaced67aec7a06d/thriftbp/prometheus.go

Reply:
Moved deprecation comment to package level & added prometheus counters.